src/core/src/Axis.cpp

```diff
@@ -92,6 +92,30 @@ namespace iDynTree
 
         return nonRotated_T_rotated;
     }
+    
+      Transform Axis::getTranslationTransform(const double dist) const
+    {
+        Transform nonTranslated_T_translated;
+
+        //No rotation
+        nonTranslated_T_translated.setRotation(Rotation::RotAxis(this->getDirection(),0.0));
```

Comment:
Please use `iDynTree::Rotation::Identity()` instead of `Rotation::RotAxis(this->getDirection(),0.0)`.